modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -6,11 +6,13 @@ Redpanda supports applying throughput throttling on both ingress and egress inde
 
 == Throughput throttling enforcement
 
+NOTE: As of v24.2, Redpanda enforces all throughput limits per broker, including client throughput.  
```

Comment:
Do you know the RC/build number in which this first appears and will be released? I've been seeing `24.2.1` and `24.2.2` and am wondering which will be accurate for the release.

Reply:
I don't think I've seen anything more specific or granular, @pgellert could you confirm?

Reply:
This is going out with the v24.2 major version release, so it is already in the beta releases v24.2.1-rc1 through v24.2.1-rc4 and the first non beta release v24.2.1 will contain the change. I think phrasing it as either v24.2 or v24.2.1 makes sense.